### PR TITLE
Change comment about adding reviews

### PR DESCRIPTION
**Pull request by Olavhaasie**
Added comment in template that states that reviews will be automatically requested.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ e.g.
 * Added topic
 -->
 
-<!-- Please don't forget to request for reviews -->
+<!-- Reviews are automatically requested after the pull request has been created. -->
```
Comment (RutgerVanBeek) [inline pull_request_template.md]:
```suggestion
<!-- Reviews are automatically requested after the pull request has been created. Only request for a review if you want a specific person to review your changes.-->
```

**Review by RutgerVanBeek — CHANGES_REQUESTED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (Roelemans) [inline pull_request_template.md]:
```suggestion
<!-- Reviewers from the software team are automatically requested after the pull request has been created. -->
```

**Review by Roelemans — CHANGES_REQUESTED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (Olavhaasie) [reply pull_request_template.md]: This is not true, other teams and people can also be requested, check the `CODEOWNERS` files

Comment (Roelemans) [reply pull_request_template.md]: Ah, I see

**Review by Joris-JW — APPROVED** [review]

**Review by Roelemans — APPROVED** [review]

**Review by RutgerVanBeek — APPROVED** [review]
Thanks!

**Comment by Roelemans** [comment]
5 reviews in 10 min :O

**Pull request merged**